src/util/buildRegex.js

```diff
@@ -0,0 +1,10 @@
+let _ = require('lodash/fp')
+
+module.exports = _.flow(
+  _.replace(/\s\s+/g, ' '),
+  _.trim,
+  _.split(' '),
```

Comment:
these first 3 parts of the flow could be replaced with  `_.words`

https://runkit.com/daedalus28/words-trimming

After that, I wonder if this is a better fit for futil 🤔 

We should also rename this from `buildRegex` to something clearer. The name was fine before because it was specific to `facet`, but now it's more general and is probably something like `toAnyWordsRegex` (although it's not just words). We have precedent of _similar_ methods like these regex utils in futil:
https://github.com/smartprocure/futil-js/blob/master/src/regex.js#L13-L31

Reply:
futil probably makes more sense so I'll get set to take it over there

Reply:
@daedalus28 going to do this: https://github.com/smartprocure/futil-js/issues/273

One note: the current code here in `contexture-mongo` `filter` does a "match all". I think we want "match any". When the Futil change is in I'll bump the version here and change it up to do that.

Reply:
We do want match all in this case - as you type more words, it should restrict the set of things you're looking for, not expand it. Excuse my terrible ascii art:

```
| "pad case" |
[ ] ipad chager
[ ] iphone case
[ ] pad cases <-- only this one should show 
```

Reply:
got it -- thanks, will do